bon-macros/src/builder/builder_gen/setters/mod.rs

```diff
@@ -491,18 +533,35 @@ impl SettersItems {
 
         let doc = |docs: &str| iter::once(syn::parse_quote!(#[doc = #docs]));
 
+        let getter_fn = member.config.getter.as_ref().map(|getter_config| Item {
```

Comment:
This is also violating single responsibility. Feel free to just copy the relevant parts of this logic into a separate `getters` module.